gui/qt/console.py

```diff
@@ -183,8 +183,6 @@ def keyPressEvent(self, event: QtGui.QKeyEvent):
             return
         elif event.key() == QtCore.Qt.Key_L and event.modifiers() == QtCore.Qt.ControlModifier:
             self.parent().clear()
-        elif event.key() == QtCore.Qt.Key_Insert and event.modifiers() == QtCore.Qt.NoModifier:
```

Comment:
Hey .. I presume you didn't delete this. Would you mind rebasing to master so that this PR doesn't undo recent commits??  Or was this intentional?

Reply:
Ah, must have missed a commit last night. One second, I'll just add it back in.

Reply:
Ha, ok. 👍